### ISSUE TITLE
As a timetable planner, I want the a timetable row to indicate why it is unclickable.

### ISSUE THREAD

**Issue by juhakolari** [HSLdevcom/jore4#1507]
The list view of a missing timetable for a route should contain the words “Ei aikataulua”. ([Figma](https://www.figma.com/file/ImSTkCqQn0nhVUtMcUm41P/Timetables-%7C-Routes-and-lines-%7C-JORE-4.0-UX?type=design&node-id=7508%3A323329&mode=design&t=NB4J4zUo9meK2QGq-1), see also below)

![Image](https://github.com/HSLdevcom/jore4/assets/77674841/d0e1eb84-4f6a-40cd-8712-4e7962c723b5)
